### PR TITLE
Add delay to tentatively fix flakyness with Docker

### PR DESCRIPTION
**Pull request by apostasie**

### CODE REVIEW

File: cmd/nerdctl/container/container_logs_test.go [hunk 1]
```diff
@@ -95,6 +95,9 @@ func TestLogsWithInheritedFlags(t *testing.T) {
 	base.Cmd("run", "-d", "--name", containerName, testutil.CommonImage,
 		"sh", "-euxc", "echo foo; echo bar").AssertOK()
 
+	// It appears this test flakes out with Docker seeing only "foo\n"
+	// Tentatively adding a pause in case this is just slow
+	time.Sleep(time.Second)
```
Comment (AkihiroSuda) [inline cmd/nerdctl/container/container_logs_test.go:100]: Seems a bug in Docker?

**Review by AkihiroSuda — APPROVED** [review]
Thanks

File: cmd/nerdctl/container/container_logs_test.go (hunk 1 above)
Comment (apostasie) [reply cmd/nerdctl/container/container_logs_test.go:100]: Maybe - if that does not help, I ll just mark it skipped for docker

**Comment by apostasie** [comment]
Similar failure happened in `TestStopWithStopSignal`.
Documenting with #3564.

There may indeed be some bug in docker logs.

**Pull request merged**